generators/app/templates/src/screens/Dashboard/screens/Home/index.tsx

```diff
@@ -13,24 +9,17 @@ import { reducer, INITIAL_STATE, actionCreators } from './reducer';
 
 function Home() {
   // Example of how to use these custom hooks
-  const foo = useSelector(state => state.foo);
+  const foo = useSelector((state) => state.foo);
   const dispatch = useDispatch();
-  const userDispatch = useUserDispatch();
-  const [, , , logoutRequest] = useLazyRequest({
-    request: logout,
-    withPostSuccess: () => {
-      userDispatch(authActions.resetUser());
-      removeCurrentUserToken();
-    }
-  });
 
   useEffect(() => {
     dispatch(actionCreators.setFoo('React'));
   }, [dispatch]);
 
   const handleLogout = () => {
-    userDispatch(authActions.logout());
-    logoutRequest(null);
+    // Perform logout request and on success add the following:
+    // userDispatch(authActions.resetUser());
+    // removeCurrentUserToken();
```

Comment:
Why did you remove this dispatch?

Reply:
Also check conflicts

Reply:
I didn't remove it, I purposedly left them commented because I thought they should be in the logout success callback. But now that I think of it, we should perform these two actions even if logout fails I suppose. So I'll uncomment them